Doc/library/asyncio-runner.rst

```diff
@@ -119,3 +119,30 @@ Runner context manager
 
       Embedded *loop* and *context* are created at the :keyword:`with` body entering
       or the first call of :meth:`run` or :meth:`get_loop`.
+
+
+Handling Keyboard Interruption
```

Comment:
@kumaraditya303 could you review this section, please?
Feel free to suggest wild rephrasing if you don't like the chapter structure.

Reply:
Sure, I have added inline suggestions.